From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yann Dirson <yann.dirson@vates.tech>
Date: Fri, 22 Aug 2025 15:06:43 +0200
Subject: [PATCH] Second "dnf install" run for crypto-policies

This is a huge hack, not easy to put in configureXcpng.
---
 repository.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/repository.py b/repository.py
index 66d05cbc..8e3c94ab 100644
--- a/repository.py
+++ b/repository.py
@@ -893,6 +893,9 @@ baseurl=%s
         targets = list(set(targets))
 
         installFromYum(targets, mounts, progress_callback, cachedir)
+        # for some reason installing at the same time as some undetermined
+        # other does not work, silently (hail DNF/RPM :sob:)
+        installFromYum(['crypto-policies'], mounts, progress_callback, cachedir)
         repos[0].enableInitrdCreation()
     finally:
         for repo in repos:
